grants/faq.md

```diff
@@ -0,0 +1,153 @@
+#### Web3 Foundation Grants Programs'
+
+# :interrobang: Frequently Asked Questions :interrobang:<!-- omit in toc -->
+
+- [🧭 Global (Open & General grants)](#-global-open--general-grants)
+- [🔓 Open Grants](#-open-grants)
+- [💰 General Grants](#-general-grants)
+- [🚚 Milestone Delivery](#-milestone-delivery)
+
+## 🧭 Global (Open & General grants)
+
+<details>
+  <summary><b>How much can I ask for?</b></summary>
+  The Open Grants program is capped at US$ 30k for initial grants, meaning a project can only apply for 30k. If the grant is successfully completed, you can apply for a follow-up grant, which is capped at 100k. The General Grants program (https://github.com/w3f/General-Grants-Program/) is capped at 100k, although higher grants are possible in exceptional cases. However, we only accept applications for the General grants program if you have a reason for the project to remain private or you can only take fiat payments.
+</details>
+
+<details>
+  <summary><b>How long does it take from application to decision?</b></summary>
+
+  Depending on the quality of the application and desirability for the ecosystem, an OPEN grant application could be approved within a week. Usually, there will be a discussion and requests for changes, additions or improvements. If no one in the committee finds the application approval-worthy, it will be closed after two weeks of inactivity. For GENERAL grants, the council convenes once a month to discuss open applications, meaning: once an editor finds your application sufficient, it'll be at most one month before it is decided upon.
```

Comment:
```suggestion
  Depending on the quality of the application and desirability for the ecosystem, an OPEN grant application could be approved within a week. Usually, there will be a discussion and requests for changes, additions or improvements. If no one in the committee finds the application approval-worthy or you don't react to our comments, it will be closed after two weeks of inactivity. For GENERAL grants, the council convenes once a month to discuss open applications, meaning: once an editor finds your application sufficiently elaborate, it'll take at most one month until a decision is made.
```